docs/releases/5.2.txt

```diff
@@ -198,6 +198,10 @@ Management Commands
   ``runserver`` is unsuitable for production. This warning can be hidden by
   setting the :envvar:`HIDE_PRODUCTION_WARNING` environment variable to
   ``"true"``.
+* The :djadmin:`shell` management command now automatically imports all models
```

Comment:
```suggestion

* The :djadmin:`shell` management command now automatically imports all models
```
(you also want to get rid of a bunch of stuff in the commit description if you can)

Reply:
This change got lost

Reply:
I'm sorry :disappointed_relieved:

Reply:
No problem :heart: